cql3/statements/alter_table_statement.cc

```diff
@@ -256,16 +262,22 @@ void alter_table_statement::drop_column(const schema& schema, data_dictionary::t
             }
         }
 
+        std::optional<api::timestamp_type> drop_timestamp;
+        if (_attrs->is_timestamp_set()) {
+            auto now = std::chrono::duration_cast<std::chrono::microseconds>(db_clock::now().time_since_epoch()).count();
+            drop_timestamp = _attrs->get_timestamp(now, options);
```

Comment:
Finally I understood what "options" is all about.... If I understand correctly, "_attrs" may only contains the "?" reference, and real value comes in options - is that right?
I'm still not sure I understand why there isn't there some earlier place in the code which can do this resolution without needing to add this "options" parameter - don't we have other statements already which need to allow bound variables and needed to do this already?

Reply:
Attributes contains the value of the timestamp if the timestamp was specified during statement creation and it needs query_options if the statement was prepared with `?`.

Schema statements are executed via query_processor::execute_schema_statement(), which executes stmt.prepare_schema_mutations(), which is the function the signature I've changed.

I see the options are not used in most of the statements but the alternative would require to rebuild how schema statements are executed and this PR is supposed to be a simple fix for a release blocking issue.